library/foundation/src/main/java/com/epmedu/animeal/foundation/dialog/AnimealQuestionDialog.kt

```diff
@@ -87,4 +87,18 @@ private fun AnimealAlertQuestionPreview() {
             onConfirm = {}
         )
     }
+}
+
+@AnimealPreview
+@Composable
+private fun AnimealAlertQuestionSingleOptionPreview() {
```

Comment:
This Preview basically functions the same as the above one, just with different strings for the title and buttons. Perhaps, we can remove it?